### ISSUE TITLE
Accessibility issue started to happen suddenly.

### ISSUE THREAD

**Issue by senihicalakadaretmez258** [desktop/desktop#22203]
### The problem

Hello, a few days ago, the github desktop started to be inaccessible for visualy impaired people. For example, it's not reading branches, commits, changed file names. Those are the latest accessibility issues which I'm facing. I'll be happy if you check it and fix this issue. Thank you!

### Release version

Version 3.5.11 (x64)

### Operating system

Windows 10

**Comment by niik** [comment]
Hi @senihicalakadaretmez258, that's odd! I've been running the app using a screenreader quite a bit these last few days as I've been addressing a11y issues and I've been unable to reproduce what you're seeing. Could you please provide some more details around your operating system, what assistive technologies you're using (preferably include version numbers) and anything else that could help us troubleshoot this, thanks!

**Comment by senihicalakadaretmez258** [comment]
Hello, I use NCVDA as screen reader (version 2303.3) but i think it's same for all versions, haven't tried that. The problem is everything was ok like one or two weeks ago, maybe when the new update came out.

**Comment by senihicalakadaretmez258** [comment]
Sorry I wanted to say NVDA

**Comment by niik** [comment]
Hi @senihicalakadaretmez258, thanks for reporting this! We've been able to reproduce this with NVDA on Windows and are working on a fix. While we investigate the root cause the only workaround I can offer is for you to download and install GitHub Desktop 3.5.8 which we've confirmed is working as intended with NVDA. The direct download link for the 3.5.8 installer is https://github.com/desktop/desktop/releases/download/release-3.5.8/GitHubDesktopSetup-x64.exe

**Comment by niik** [comment]
We believe this is the same issue as was reported in https://github.com/desktop/desktop/issues/22219 so I'm gonna close this issue in favor of it. Please subscribe to that issue to follow our progress in addressing this issue.

**Comment by niik** [comment]
I wanted to follow up and say that when installing an older version GitHub Desktop will attempt to auto-update to the latest version. It won't do so though as long as the app stays running. If you have to restart the application it will auto-update to the latest version on launch and you'll have to run the installer once again.

**Comment by tidy-dev** [comment]
@senihicalakadaretmez258 The fix has been shipped in [Release 3.5.12.](https://github.com/desktop/desktop/releases/tag/release-3.5.12) Thank you again for the report!

**Comment by senihicalakadaretmez258** [comment]
Thank you for your good work. I'm happy that it has been fixed. I would like to congratulate you and wish you success in your work. Have a good day.
